src/main/java/medicaldoctor/setup/InitTestDatabase.java

```diff
@@ -103,7 +103,6 @@ private static void insertTestRecords() {
         user.setType(UserType.ADMIN);
         user.setDepartment("Administration");
         user.setOfficeNum(145);
-        user.setEmail();
```

Comment:
it should still be set, you can have a local EmailGenerator to set the values,
maybe even put a loop at the end:
```java
for(User user : somelist){
    user.setEmail(emailGenerator.generateEmail(user.getUsername()));
}
```